kim/kim-ldap/src/main/java/org/kuali/rice/kim/dao/impl/LdapPrincipalDaoImpl.java

```diff
@@ -306,7 +306,7 @@ public Entity getEntityByEmployeeId(String employeeId) {
         }
 
         Map<String, Object> criteria = new HashMap();
-        String key = getLdapAttribute(getKimConstants().getKimLdapIdProperty());
+        String key = getKimConstants().getKimLdapIdProperty();
```

Comment:
Correct, that line removes a "translation" of a property that was already translated to LDAP terms.